### PR TITLE
Fix NDC coordinates for Vulkan in Advanced post-processing

### PR DESCRIPTION
**Pull request by Calinou**
This closes https://github.com/godotengine/godot-docs/issues/5276.

**Note:** Don't cherry-pick to 3.x branches.

### CODE REVIEW

**Review by clayjohn — COMMENTED** [review]
Line 100 needs to change as well.

**Comment by Calinou** [comment]
> Line 100 needs to change as well.

Fixed.

File: tutorials/shaders/advanced_postprocessing.rst [hunk 1]
```diff
@@ -177,7 +182,7 @@ Now, attach a script to the MeshInstance and use the following code:
     mesh.add_surface_from_arrays(Mesh.PRIMITIVE_TRIANGLES, mesh_array)
 
 .. note:: The triangle is specified in normalized device coordinates. Recall, NDC run
-          from ``-1`` to ``1`` in both the ``x`` and ``y`` directions. This makes the screen
+          from ``0.0`` to ``1.0`` in both the ``x`` and ``y`` directions. This makes the screen
```
Comment (clayjohn) [inline tutorials/shaders/advanced_postprocessing.rst:185]:
```suggestion
          from ``-1.0`` to ``1.0`` in both the ``x`` and ``y`` directions. This makes the screen
```

File: tutorials/shaders/advanced_postprocessing.rst [hunk 2]
```diff
@@ -97,15 +97,20 @@ inverse of the projection matrix, which in Godot, is accessible with the variabl
 ``INV_PROJECTION_MATRIX``.
 
 Firstly, take the screen space coordinates and transform them into normalized device
-coordinates (NDC). NDC run from ``-1`` to ``1``, similar to clip space coordinates.
+coordinates (NDC). NDC run from ``0.0`` to ``1.0``, similar to clip space coordinates.
```
Comment (clayjohn) [inline tutorials/shaders/advanced_postprocessing.rst:100]:
```suggestion
coordinates (NDC). NDC run from ``-1.0`` to ``1.0`` in ``x`` and ``y`` directions and from ``0.0`` to ``1.0`` in the ``z`` direction when using the Vulkan backend.
```

**Comment by Calinou** [comment]
Should this instance of `-1` be changed too? https://github.com/Calinou/godot-docs/blob/8e640487b8b31e1ffb0d49bf92a21fde8b68588c/tutorials/shaders/advanced_postprocessing.rst#L45

File: tutorials/shaders/advanced_postprocessing.rst (hunk 1 above)
Comment (Calinou) [reply tutorials/shaders/advanced_postprocessing.rst]: Is it actually between -1.0 and 1.0? I thought that was in OpenGL.

**Comment by clayjohn** [comment]
I don't think so. I think we still use a clip space of -1 to 1 in all three dimensions

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Thank you! And thanks to clayjohn for reviewing :)

**Pull request merged**